### PR TITLE
feat: migrate cli fetch

### PR DESCRIPTION
**Pull request by loks0n**
- Migrate CLI from axios to fetch
- Refactor commands.js.twig to match patterns from Node and Web templates

### CODE REVIEW

**Comment by stnguyen90** [comment]
@loks0n, would you be able to look into why the node tests are failing?

File: templates/cli/lib/parser.js.twig [hunk 1]
```diff
@@ -163,6 +163,7 @@ const commandDescriptions = {
     "locale": `The locale command allows you to customize your app based on your users' location.`,
     "projects": `The projects command allows you to view, create and manage your {{ spec.title|caseUcfirst }} projects.`,
     "storage": `The storage command allows you to manage your project files.`,
+    "messaging": `The messaging command allows you to manage and send messages to your users`,
```
Comment (christyjacob4) [inline templates/cli/lib/parser.js.twig:166]: we have to remove 1.5 related items from here. The fixed CLI needs a release before 1.5

File: templates/cli/lib/commands/generic.js.twig [hunk 2]
```diff
@@ -6,7 +6,7 @@ const { globalConfig, localConfig } = require("../config");
 const { actionRunner, success, parseBool, commandDescriptions, log, parse } = require("../parser");
 {% if sdk.test != "true" %}
 const { questionsLogin } = require("../questions");
-const { accountCreateEmailSession, accountDeleteSession } = require("./account");
+const { accountCreateEmailPasswordSession, accountDeleteSession } = require("./account");
```
Comment (christyjacob4) [inline templates/cli/lib/commands/generic.js.twig:9]: Is this for 1.5 ? If not lets remove it and change the base branch to main

**Comment by PineappleIOnic** [comment]
This PR has been refactored to master under #771 and can be closed if needed

**Pull request closed without merging**